src/controllers/channel.controller.js

```diff
@@ -140,4 +140,33 @@ exports.AddUserToChannel = async (req, res) => {
     {
         return response_500(res, "Error removing user from channel", err);
     }
+}
+
+exports.removeUserFromChannel = async (req,res)=>
+{
+    try {
+        const{user,channel} = req.body;
+        const userToBeRemovedId = req.params.id;
```

Comment:
req.params.id      ->   req.body.userId

Reply:
But won't that be the user calling the API? The admin? Or we are having a middleware which fetches the user to be removed for us and passes it in req.body?